### PR TITLE
June newsletter

### PR DESCRIPTION
**Pull request by maelle**

### CODE REVIEW

File: content/blog/2023-06-23-ropensci-news-digest-june-2023/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,691 @@
...
+### Statistical software review now completely integrated with standard review process
+
+Statistical software review has finally reached a stable and mature state, with all organizational processes now entirely integrated with the standard review process.
+Developers submitting software should notice little difference, except hopefully faster initial processing of submissions.
+Each submission is now smoothly handled by members of our team of statistical software editors:
+
+-   [Adam Sparks](/author/adam-sparks/)
+-   [Emi Tanaka](https://emitanaka.org/)
+-   [Jouni Helske](/author/jouni-helske/)
+-   [Paula Moraga](http://www.paulamoraga.com/)
```
Comment (steffilazerte) [inline content/blog/2023-06-23-ropensci-news-digest-june-2023/index.Rmd:67]:
```suggestion
-   [Paula Moraga](https://www.paulamoraga.com/)
```

File: content/blog/2023-06-23-ropensci-news-digest-june-2023/index.Rmd [hunk 2]
```diff
@@ -0,0 +1,691 @@
...
+-   [Rebecca Killick](/author/rebecca-killick/)
+-   [Toby Hocking](http://tdhock.github.io/)
```
Comment (steffilazerte) [inline content/blog/2023-06-23-ropensci-news-digest-june-2023/index.Rmd:69]:
```suggestion
-   [Toby Hocking](https://tdhock.github.io/)
```

File: content/blog/2023-06-23-ropensci-news-digest-june-2023/index.Rmd [hunk 3]
```diff
@@ -0,0 +1,691 @@
...
+
+We current accept software in the eight categories described in our [*Statistical Software Development Guide*](https://stats-devguide.ropensci.org/standards.html), and are still working on integrating the two additional categories of statistical workflow and network software.
```
Comment (steffilazerte) [inline content/blog/2023-06-23-ropensci-news-digest-june-2023/index.Rmd:71]:
```suggestion
We currently accept software in the eight categories described in our [*Statistical Software Development Guide*](https://stats-devguide.ropensci.org/standards.html), and are still working on integrating the two additional categories of Statistical Workflow and Network Software.
```

File: content/blog/2023-06-23-ropensci-news-digest-june-2023/index.Rmd [hunk 4]
```diff
@@ -0,0 +1,691 @@
...
+We encourage anybody thinking about submitting to contact any one of the editors, or the two leaders of the software review team, [Noam Ross](/author/noam-ross/) and [Mark Padgham](/author/mark-padgham/), at any time, including anybody interested in submitting software in the upcoming categories of workflow or network software.
+
+### rOpenSci's Communication Channels for Safe and Friendly Exchange
+
+As [we anticipated in November 2022](/blog/2022/11/16/mastodon-en) the changes implemented by Twitter have generated a less safe and friendly space for our community.  
+That is why from _June 2023_, we will stop interacting on this platform.
```
Comment (steffilazerte) [inline content/blog/2023-06-23-ropensci-news-digest-june-2023/index.Rmd:77]:
```suggestion
As [we anticipated in November 2022](/blog/2022/11/16/mastodon-en) the changes implemented by Twitter have generated a less safe and friendly space for our community.
That is why from _June 2023_, we will stop interacting on this platform. 
```

File: content/blog/2023-06-23-ropensci-news-digest-june-2023/index.Rmd [hunk 5]
```diff
@@ -0,0 +1,691 @@
...
+Hosted by Steffi LaZerte and various community hosts. 
+Everyone welcome. 
+No RSVP needed. 
+Consult our [Events](/events) page to find your local time and how to join.
+
+* Tuesday, July 4th, 14:00 European Central / 12:00 UTC ["Create/Update your 'Happy File'/'Brag Document'!"](/events/coworking-2023-07/) *Hosted by [Maëlle Salmon](/author/maëlle-salmon) and [Steffi LaZerte](/author/steffi-lazerte/)*
+    - Explore what goes into a [‘Happy File’](https://twitter.com/JennyBryan/status/1582862196870373377)/[‘Brag Document’](https://jvns.ca/blog/brag-documents/) and why you need one.
+    - Start collecting items to add to your ‘Happy File’.
+    
+- Tuesday, August 1st, 09:00 Americas Pacific (16:00 UTC), [Spatial data in R](/events/coworking-2023-08/) *Hosted by [Mike Mahoney](/author/mike-mahoney) and [Steffi LaZerte](/author/steffi-lazerte/)*
```
Comment (steffilazerte) [inline content/blog/2023-06-23-ropensci-news-digest-june-2023/index.Rmd:114]:
```suggestion
- Tuesday, August 1st, 09:00 Americas Pacific (16:00 UTC), ["Spatial data in R"](/events/coworking-2023-08/) *Hosted by [Mike Mahoney](/author/mike-mahoney) and [Steffi LaZerte](/author/steffi-lazerte/)*
```

File: content/blog/2023-06-23-ropensci-news-digest-june-2023/index.Rmd [hunk 6]
```diff
@@ -0,0 +1,691 @@
...
+### How to produce code hints for users?
+
+Sometimes your code can infer what the next user step should be, or what the next _possible_ user _steps_ could be.
+How to make it easy for the user to run these hints?
+
+Here's some inspiration:
+
+* You could use [`cli::ui_todo()`](https://usethis.r-lib.org/reference/ui.html), as done in usethis. By the way if your package is using `ui_` functions from usethis, don't miss the cli article on how to [migrate your code to using cli instead](https://cli.r-lib.org/articles/usethis-ui.html), as that's what's recommended. 
+
+* Also in the cli package, a special kind of links that runs the code if it is clicked on, in RStudio IDE: ["Click to run code"](https://cli.r-lib.org/reference/links.html?q=run#click-to-run-code). You might know that from running snapshot tests with testthat, and getting told to review/accept them with these special links.
```
Comment (steffilazerte) [inline content/blog/2023-06-23-ropensci-news-digest-june-2023/index.Rmd:662]:
```suggestion
* Also in the cli package, a you can provide a special kind of link that runs the code if clicked in the RStudio IDE: ["Click to run code"](https://cli.r-lib.org/reference/links.html?q=run#click-to-run-code). You might recognize this from running snapshot tests with testthat, and receiving the message to review/accept them with these special links.
```

File: content/blog/2023-06-23-ropensci-news-digest-june-2023/index.Rmd [hunk 7]
```diff
@@ -0,0 +1,691 @@
...
+Some useful tips for R package developers. :eyes:
+
+### How to produce code hints for users?
+
+Sometimes your code can infer what the next user step should be, or what the next _possible_ user _steps_ could be.
+How to make it easy for the user to run these hints?
+
+Here's some inspiration:
+
+* You could use [`cli::ui_todo()`](https://usethis.r-lib.org/reference/ui.html), as done in usethis. By the way if your package is using `ui_` functions from usethis, don't miss the cli article on how to [migrate your code to using cli instead](https://cli.r-lib.org/articles/usethis-ui.html), as that's what's recommended.
```
Comment (steffilazerte) [inline content/blog/2023-06-23-ropensci-news-digest-june-2023/index.Rmd:660]:
```suggestion
* You could use [`cli::ui_todo()`](https://usethis.r-lib.org/reference/ui.html), as done in usethis. By the way if your package is using `ui_` functions from usethis, don't miss the cli article on how to [migrate your code to using cli instead](https://cli.r-lib.org/articles/usethis-ui.html), as that's what's currently recommended. 
```

File: content/blog/2023-06-23-ropensci-news-digest-june-2023/index.Rmd [hunk 8]
```diff
@@ -0,0 +1,691 @@
...
+
+* You could use [`cli::ui_todo()`](https://usethis.r-lib.org/reference/ui.html), as done in usethis. By the way if your package is using `ui_` functions from usethis, don't miss the cli article on how to [migrate your code to using cli instead](https://cli.r-lib.org/articles/usethis-ui.html), as that's what's recommended. 
+
+* Also in the cli package, a special kind of links that runs the code if it is clicked on, in RStudio IDE: ["Click to run code"](https://cli.r-lib.org/reference/links.html?q=run#click-to-run-code). You might know that from running snapshot tests with testthat, and getting told to review/accept them with these special links.
+
+```r
+cli::cli_text("Run {.run praise::praise()} now!")
+```
+
+* If your code is run in RStudio IDE (checkable with [`rstudioapi::isAvailable()`](https://rstudio.github.io/rstudioapi/reference/isAvailable.html)), you could use [`rstudioapi::sendToConsole()`](https://rstudio.github.io/rstudioapi/reference/sendToConsole.html). [Example](https://github.com/cynkra/fledge/blob/520b9b2e36da7bc58136378407f1f4454bfbeadf/R/finalize-version.R#L63-L80=).
```
Comment (steffilazerte) [inline content/blog/2023-06-23-ropensci-news-digest-june-2023/index.Rmd:668]: I'm not sure what the purpose of doing this is... does my suggestion make sense?
```suggestion
* If your code is run in RStudio IDE (checkable with [`rstudioapi::isAvailable()`](https://rstudio.github.io/rstudioapi/reference/isAvailable.html)), you could use [`rstudioapi::sendToConsole()`](https://rstudio.github.io/rstudioapi/reference/sendToConsole.html) to send example code to the console for the user to consider ([example](https://github.com/cynkra/fledge/blob/520b9b2e36da7bc58136378407f1f4454bfbeadf/R/finalize-version.R#L63-L80=)).
```

**Review by steffilazerte — COMMENTED** [review]
Hi @maelle, looks good, thanks!

* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md

**Comment by maelle** [comment]
Thank you @steffilazerte!!

**Pull request merged**